### PR TITLE
Adds better documentation for watchman, resolves #3687

### LINKED ISSUE #3687: #watchman section of site should explain a bit more
http://www.ember-cli.com/#watchman

something with more word smithing ability then me should likely do this.

* TL:DR OSX filewatching has some bugs, that can result in it easily entering a broken state
* the built in file watching in Node `NodeWatcher` has some issue, especially around observer large trees
* watchman aims to make file-watching work on mega-massive trees of files.  (facebook uses it for there devs, on some massive source trees)

### PR DESCRIPTION
**Pull request by runspired**

### CODE REVIEW

File: _posts/2014-04-03-getting-started.md [hunk 1]
```diff
@@ -47,15 +47,47 @@ This will give you access to the `bower` command-line runner.
...
+following warning.
+
+{% highlight bash %}
+invalid watchman found, version: [2.9.8] did not satisfy [^3.0.0], falling back to NodeWatcher
+{% endhighlight %}
+
+The following command will let you inspect which watchmans you have.
+
+{% highlight bash %}
+which -a
```
Comment (pangratz) [inline _posts/2014-04-03-getting-started.md]: This should be `which -a watchman`, right?

Comment (stefanpenner) [inline _posts/2014-04-03-getting-started.md]: we should also hint to what the correct path may look like

File: _posts/2014-04-03-getting-started.md [hunk 2]
```diff
@@ -47,15 +47,47 @@ This will give you access to the `bower` command-line runner.
...
+{% endhighlight %}
+
+If you have installed the wrong watchman, you can use npm uninstall to remove it.
```
Comment (stefanpenner) [inline _posts/2014-04-03-getting-started.md]: we should give the exact command which is likely `npm uninstall -g watchman`

**Comment by stefanpenner** [comment]
other then the 2 comments LGTM

File: _posts/2014-04-03-getting-started.md [hunk 3]
```diff
@@ -47,15 +47,47 @@ This will give you access to the `bower` command-line runner.
...
+The following command will let you inspect which watchmans you have.
+
+{% highlight bash %}
+which -a watchman
+{% endhighlight %}
+
+If you have installed the wrong watchman, you can use npm uninstall to remove it.
+
+{% highlight bash %}
+npm uninstall -g watchman
```
Comment (runspired) [inline _posts/2014-04-03-getting-started.md]: @stefanpenner that was already there :D

File: _posts/2014-04-03-getting-started.md (hunk 1 above)
Comment (runspired) [inline _posts/2014-04-03-getting-started.md]: @pangratz It should be, have added it!

**Comment by runspired** [comment]
@stefanpenner @pangratz made the change

**Comment by stefanpenner** [comment]
rad, thanks!

**Pull request merged**